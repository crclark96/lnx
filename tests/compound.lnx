ax = 5;
ax + bx - dx;
ax = dx >> 5;
