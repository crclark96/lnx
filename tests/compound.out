mov ax, 5
add ax, bx
sub ax, dx
mov ax, dx
ror ax, 5
